src/sections/DiscoveryHero/DiscoveryHero.scss

```diff
@@ -0,0 +1,26 @@
+.discovery-hero {
+    width: 100%;
+
+    &__box {
+        background-color: theme-color("accent-2");
+        @include if-desktop {
+            padding: 0 140px;
+        }
+    }
+
+    &__content {
+        background-image: url('/assets/img/DiscoveryMountain.jpg');
+        background-size: cover;
+        padding: 90px spacer(small-xl);
+
+        @include if-tablet {
+            padding: 110px 170px;
```

Comment:
kazkaip interiasnai tu tuos paddingus naudoji... 
Pabandyk display flex, ir centruot horizontaliai ir vertikaliai.

Reply:
Padingai man nusako visos dezes dydi ir sutvarko man, kur tekstas turi persokti i kita eilute. Jei daryciau centralizuota, tada reiktu nustatyti tikslius ir height ir width ir componentas nebebutu toks flexible mano nuomone. Todel ir su paddingais taip ir padariau :D

Reply:
1. Tai width: 100% musu architekturoj horzontaliai sucentruot gali su flexbox.
2. Vertikaliai irgi gali su centruot su flexboxu
3. paddingas gali likt kaip soft cussion jeigu taptu maziausio imanomo dydzio, tokiu atveju neuzlystu uz paddingo
4. kadangi scrollinam vertikaliai, tai height gali nurodyt 😄

Reply:
Nu gerai yra galimybiu, ir man rodos vis kartais pamirstu width 100% irasyt :D ir del to tikrai esu galvojes, kodel man cia nesicentruoja :DD